### PR TITLE
Fix failure to launch from directory with space in it

### PR DESCRIPTION
**Pull request by thomash-acinq**
Fixes https://github.com/ACINQ/eclair/issues/2885

### CODE REVIEW

**Comment by t-bast** [comment]
Seems like the linked issue mentions that this was tested on macOS, was this tested on linux and windows as well?

File: eclair-node/src/main/resources/eclair-node.sh [hunk 1]
```diff
@@ -349,7 +349,7 @@ declare -r real_script_path="$(realpath "$0")"
 declare -r app_home="$(realpath "$(dirname "$real_script_path")")"
 declare -r lib_dir="$(realpath "${app_home:0:${#app_home}-4}/lib")" # {app_home:0:${#app_home}-4} transforms ../bin in ../
 declare -a app_mainclass=("fr.acinq.eclair.Boot")
-declare -a app_entrypoint=$(ls $lib_dir |grep eclair-node) # TODO: improve this
+declare -a app_entrypoint=$(ls "$lib_dir" |grep eclair-node) # TODO: improve this
```
Comment (pm47) [inline eclair-node/src/main/resources/eclair-node.sh:352]:
```suggestion
declare -a app_entrypoint=$(ls "$lib_dir" | grep eclair-node) # TODO: improve this
```

**Review by pm47 — COMMENTED** [review]
Testing intellij github integration

**Review by pm47 — DISMISSED** [review]
> @pm47 can probably do the windows test.

I confirm that it works on windows

**Review by pm47 — APPROVED** [review]

**Pull request merged**